databases/librasterlite2/Portfile

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                librasterlite2
+version             1.1.0-beta1
+revision            0
+
+categories          databases gis
+
+license             {MPL-1.1 GPL-2+ LGPL-2.1+}
+maintainers         {yahoo.com:n_larsson @nilason} openmaintainer
+
+description         Library that stores and retrieves raster coverages using a SpatiaLite
+long_description    ${name} is an open source library that stores and retrieves huge \
+                    raster coverages using a SpatiaLite DBMS.
+
+homepage            https://www.gaia-gis.it/fossil/${name}/index
+master_sites        https://www.gaia-gis.it/gaia-sins/${name}-sources
+distname            ${name}-${version}
+
+checksums           rmd160  b8a8df1838d76b35a806b5c4e032c800366f3ca3 \
+                    sha256  f7284cdfc07ad343a314e4878df0300874b0145d9d331b063b096b482e7e44f4 \
+                    size    17865163
+
+patchfiles-append   patch-r2gif_c.diff
```

Comment:
where do these patches come from? If from upstream, please add a link; otherwise please make sure to submit them.

Reply:
added links to tickets in the patch files